spec/system/page_objects/components/sidebar.rb

```diff
@@ -1,138 +0,0 @@
-# frozen_string_literal: true
-
-module PageObjects
-  module Components
-    class Sidebar < PageObjects::Components::Base
```

Comment:
I split this up into `PageObjects::Components::NavigationMenu::Sidebar` and `PageObjects::Components::NavigationMenu::HeaderDropdown` since the interactions are different. Common methods are extracted into ``PageObjects::Components::NavigationMenu::Base``

Reply:
Yes works for me :+1: